src/App.js

```diff
@@ -1,9 +1,78 @@
 import React from 'react';
+import peopleFromServer from './people';
+import PeopleFilter from './PeopleFilter';
+import PeopleTable from './PeopleTable';
+import { debounce } from './debounce';
+import './App.css';
 
-const App = () => (
-  <div className="App">
-    <h1>People table</h1>
-  </div>
+const preparedPeople = peopleFromServer.map(
+  (person, index) => (
+    {
+      ...person,
+      id: index + 1,
+      age: person.died - person.born,
+      century: Math.ceil(person.died / 100),
+    }
+  )
 );
 
+class App extends React.Component {
+  state = {
+    people: [...preparedPeople],
+    filter: '',
+    isSortedBy: 'id',
+  }
+
+  filterPeople = debounce((input) => {
+    this.setState({ filter: input.trim().toLowerCase() });
+  }, 500);
+
+  sortPeopleBy = (sortBy) => {
+    if (sortBy === this.state.isSortedBy) {
```

Comment:
делай деструктуриазацию state
```suggestion
    const { isSortedBy } = this.state
    if (sortBy === isSortedBy) {
```